app/code/Magento/Catalog/etc/db_schema_whitelist.json

```
@@ -1122,5 +1124,20 @@
             "CATALOG_PRODUCT_FRONTEND_ACTION_VISITOR_ID_PRODUCT_ID_TYPE_ID": true,
             "CATALOG_PRODUCT_FRONTEND_ACTION_CUSTOMER_ID_PRODUCT_ID_TYPE_ID": true
         }
+    },
+    "catalog_compare_list": {
+        "column": {
+            "list_id": true,
+            "list_id_mask": true,
+            "customer_id": true
+        },
+        "index": {
+            "CATALOG_COMPARE_LIST_LIST_ID_MASK": true,
+            "CATALOG_COMPARE_LIST_CUSTOMER_ID": true
+        },
+        "constraint": {
+            "PRIMARY": true,
+            "CATALOG_COMPARE_LIST_CUSTOMER_ID_CUSTOMER_ENTITY_ENTITY_ID": true
+        }
     }
-}
+}
```

Hello @Usik2203 
Thank you for your contribution

Please don't forget to add a new line at end of file
```suggestion
}

```

---

This file was generated by Magento . 
It seems bad idea add empty line  at end of file.
Thanks